pennylane/wires.py

```diff
@@ -98,8 +109,41 @@ class Wires(Sequence):
         For example, running ``qml.RX(1.1, qml.numpy.array(0))`` on a device initialized with ``wires=[0]``
         will fail because ``qml.numpy.array(0)`` does not exist in the device's wire map.
 
+    .. note::
+
+        Unlike ``NumPy`` arrays, 0-dimensional arrays in ``pennylane.numpy`` are hashable.
+
     Args:
          wires (Any): the wire label(s)
+
+    Raises:
+        WireError: If the wire labels are not hashable or if the wire labels are not unique.
```

Comment:
The class itself doesn't raise this error in `init()` does it? I think it's a bit strange to say that `Wires()` raises an error when it's actually `_process()` that raises a `WireError` when given wire labels that are not hashable/unique.